sdl-core/src/main/scala/io/smartdatalake/app/GlobalConfig.scala

```diff
@@ -71,6 +78,7 @@ case class GlobalConfig(kryoClasses: Option[Seq[String]] = None
                         , pythonUDFs: Option[Map[String, PythonUDFCreatorConfig]] = None
                         , secretProviders: Option[Map[String, SecretProviderConfig]] = None
                         , allowOverwriteAllPartitionsWithoutPartitionValues: Seq[DataObjectId] = Seq()
+                        , allowAsRecursiveInput: Seq[DataObjectId] = Seq()
```

Comment:
```suggestion
                        , recursiveInputWhitelist: Seq[DataObjectId] = Seq()
```
or
```suggestion
                        , recursiveInputAllowedWhitelist: Seq[DataObjectId] = Seq()
```
With whitelist in the variable name you get the overall idea quicker without needing the docs

Reply:
Maybe, but we should avoid White/Blacklist... and allowAsRecursiveInput is similar to the previous entry allowOverwriteAllPartitionsWithoutPartitionValues. I suggest to keep it allowAsRecursiveInput.